webapp/components/LoginForm/LoginForm.vue

```diff
@@ -11,7 +11,11 @@
         </a>
       </template>
       <h2 class="title">{{ $t('login.login') }}</h2>
-      <form :disabled="pending" @submit.prevent="onSubmit">
+      <form 
+          ref="form"
+          :disabled="pending" @submit.prevent="onSubmit" 
+          @keydown.caps-lock="capsLock"
```

Comment:
When playing around I felt that
```
@keydown="capsLock"
@keyup="capsLock"
```
Has a much better behavior because we check on every key event.